cc.cubocore.CoreTime.yml

```diff
@@ -35,6 +35,6 @@ modules:
     sources:
       - type: git
         url: https://gitlab.com/cubocore/coreapps/coretime.git
-        commit: ed06e33cde531d49e277a10512e86bef34994108
+        commit: e5974464bb0934ed72db728676251e0eb96498f1
```

Comment:
I will note that retagging is a bad practice. (it's not specific to us)

Reply:
I know, 
what's the problem with the commit number here? I updated it to the latest.